package.json

```diff
@@ -31,16 +31,6 @@
     "node": ">= 14",
     "npm": ">=6.14.8"
   },
-  "nodemonConfig": {
```

Comment:
this was being overriden by the nodemon.json file so have removed this and updated the json config file.